qa/L0_model_config/autofill_platform_success/python/empty_config/config.pbtxt

```diff
@@ -0,0 +1 @@
+backend: "python"
```

Comment:
You can truly have empty config.pbtxt with Iman's change here: https://github.com/triton-inference-server/python_backend/pull/138#discussion_r868257846